packages/auth/tsconfig.lib.prod.json

```diff
@@ -2,5 +2,8 @@
   "extends": "./tsconfig.lib.json",
   "compilerOptions": {
     "declarationMap": false
+  },
+  "angularCompilerOptions": {
+    "compilationMode": "partial"
```

Comment:
This is the [suggested mode for distribution](https://angular.io/guide/creating-libraries#publishing-libraries) by the Angular team.